### PR TITLE
Fix/vbox on buster

### PR DESCRIPTION
**Pull request by neolynx**

### CODE REVIEW

**Review by szakalboss — APPROVED** [review]

File: molior-deploy [hunk 1]
```diff
@@ -470,13 +466,21 @@ mount_fs_devices()
   done
   cp -a /dev/dm-* $target/dev
   cp -a /dev/mapper $target/dev/
+  if [ -d "/run/udev/" ]; then
```
Comment (randombenj) [inline molior-deploy:470]: Why is this needed in buster?

File: molior-deploy [hunk 2]
```diff
@@ -902,8 +906,9 @@ install_grub_bootloader()
     sed -i "s%^GRUB_CMDLINE_LINUX=.*$%GRUB_CMDLINE_LINUX=\"$LINUX_CMDLINE\"%" $target/etc/default/grub
   fi
   mkdir -p $target/dev/disk/by-uuid
-  touch $target/dev/disk/by-uuid/`blkid  -o value -s UUID $rootpart`
+  cp -a /dev/disk/by-uuid/`blkid  -o value -s UUID $rootpart` $target/dev/disk/by-uuid/
```
Comment (randombenj) [inline molior-deploy:910]: https://github.com/koalaman/shellcheck/wiki/SC2006

File: molior-deploy (hunk 1 above)
Comment (neolynx) [reply molior-deploy:470]: grub-install does not work on buster without this...

will add comment, thanks

File: molior-deploy (hunk 2 above)
Comment (neolynx) [reply molior-deploy:910]: we need a separate PR for this, changing it everywhere. let's be consistent for now...
